### ISSUE TITLE
Allow status and importance icons to use shapes as well as colours

### ISSUE THREAD

**Issue by vkbo** [vkbo/novelWriter#1786]
An idea has been submitted by email to allow for the user defined colour icons used for status and importance flags to also be available as shapes. This makes them easier to distinguish for people with reduced or variations in colour vision.

This could easily be implemented by a small data class with pre-generated SVG content of common shapes where a custom colour can be added. It would require adding a shape value to the status/importance settings and in the XML, but that should be easy enough.

**Comment by vkbo** [comment]
Been playing with this feature, and I've added code that can generate 7 different shapes of icons.

![image](https://github.com/vkbo/novelWriter/assets/1619840/80b3e0a5-f3e1-4f82-ad6f-d38ef57f739c)
